Lib/defcon/objects/image.py

```diff
@@ -44,12 +44,8 @@ def __init__(self, glyph=None, imageDict=None):
         self._layer = None
         self._glyph = None
         self.glyph = glyph
-        super(Image, self).__init__()
+        super(Image, self).__init__(imageDict)
         self.beginSelfNotificationObservation()
-        self["fileName"] = None
```

Comment:
the `__len__` method below does `self["fileName"]` and crashes with `KeyError` (see failing tests).